wordpress-mobile/WordPress-iOS
# Async Posting: Show progress / upload status in Post cells

Fixes #8685. This PR hooks up the progress UI previously added to the Post list cells with progress reporting from the `PostCoordinator` / `MediaCoordinator`.

The cells should reflect the current upload progress of successful uploads (published / drafts / scheduled):

![async-posting-1](https://user-images.githubusercontent.com/4780/38039664-aa285048-32a5-11e8-9ea1-eac35088da38.gif)

As well as failures (and allow retry):

![async-posting-2](https://user-images.githubusercontent.com/4780/38039670-ac5aabcc-32a5-11e8-98c3-4795d4528ce3.gif)

**To test:**

* This one could do with quite a bit of testing! Let's check that the status is current for posts with media, without media, drafts, scheduled posts, and failures during the upload process.
* Be sure to check failures of the post upload itself, as well as media failures.
* Check that cells show the correct status when they're dequeued / reused.
